flow/scripts/report_metrics.tcl

```diff
@@ -20,6 +20,27 @@ proc report_metrics { when {include_erc true} } {
     puts "--------------------------------------------------------------------------"
     report_check_types -max_slew -max_capacitance -max_fanout -violators
 
+    if {[sta::max_slew_check_limit] < 1e30} {
```

Comment:
If you conditionalize the metric the code downstream has to be conditional also. Why don't you just let the big numbers go through?

Reply:
So that the metric meaning stays consistent.
For example, if the fanout limit is not set:
- `sta::max_fanout_check_limit` returns 1e30
- `sta::max_fanout_check_slack_limit` returns 1
While this would probably be fine for CI, the number does not convey the correct information, i.e., the design does not have a limit set.

Reply:
I claim it does convey the correct information: 1.0 (100%) says the slack is the same as the limit.
Consider a max fanout of 10. If the slack is 10 (ie, the fanout is zero) the slack/limit ratio is 1.0,
the same as the case you are suppressing.